### PR TITLE
Add boba mainnet deployment

### PR DESCRIPTION
**Pull request by boyuan-chen**
Update `boba-chain-ops` to support the boba mainnet migration. The predeploy contract address `0x4200..023` will not be pre-deployed with the `boba` token on the mainnet. The crawler tool can now retrieve the ETH allowance from Boba v1. The mainnet proxy contracts have been deployed.

### CODE REVIEW

File: boba-chain-ops/crossdomain/params.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var ParamsByChainID = map[int]*Params{
 	288: {
 		// Calculated at checkpoint block=1,000,000 - there is a
 		// small overcommittment of the OVM_ETH TotalSupply
-		new(big.Int).SetInt64(-94819327096614),
+		new(big.Int),
```
Comment (jyellick) [inline boba-chain-ops/crossdomain/params.go]: Does this comment need to be updated? It's unclear why this is different now

**Review by jyellick — APPROVED** [review]
Just to confirm my understanding of the new allowances code:

It looks like we are essentially collecting all accounts that have ever approved an allowance of eth to another address.  But, we ignore the actual allowance amount and instead look in the contract storage at the time of migration to determine the remaining allowance amount if any?

File: boba-chain-ops/crossdomain/params.go (hunk 1 above)
Comment (boyuan-chen) [reply boba-chain-ops/crossdomain/params.go]: Yes, the comment is removed.

**Comment by boyuan-chen** [comment]
> Just to confirm my understanding of the new allowances code:
> 
> It looks like we are essentially collecting all accounts that have ever approved an allowance of eth to another address. But, we ignore the actual allowance amount and instead look in the contract storage at the time of migration to determine the remaining allowance amount if any?

Yes, we will generate two tables called `eth-addresses.json` and `eth-allowances.json`. These two tables should be able to generate all slot keys of the `OVM_ETH` contract. If we miss any keys, the process will return an error.

**Comment by boyuan-chen** [comment]
@jyellick The ETH offset should be added back. The total amount of ETH is not equal to the sum of all addresses' balances. It is offset by a small amount.
```
boba-migrate-1  | EROR[03-28|19:57:40.975] supply mismatch                          migrated=2065936900010813248163 supply=2065936805191486151549 delta=-94819327096614 exp_delta=0
```

**Review by mmontour1306 — APPROVED** [review]

**Pull request merged**